wiki/People/Users_with_unique_titles/en.md

```diff
@@ -218,56 +218,12 @@ The [osu!trivium Quiz contests](/wiki/Contests/osu!trivium_Quiz) award its winni
 - ::{ flag=US }:: [Codazen](https://osu.ppy.sh/users/18291144)
 - ::{ flag=MY }:: [squidstain](https://osu.ppy.sh/users/11073207)
 
-## Featured Artists
+## osu!musician
```

Comment:
move down to above hall of famers perhaps

(hmm the order of some of these headings is quite wack)

Reply:
order is quite scuffed yea, however i like how osu!musician is currently next to osu!remix and osu!trivium for now

headings order in the entire page could be addressed later